assets/airports/cyow.json

```diff
@@ -0,0 +1,2440 @@
+                ["N45.447353100", "W75.685040700"],
+                ["N45.447085000", "W75.684834600"],
+                ["N45.446809900", "W75.684648800"],
+                ["N45.446528200", "W75.684483700"],
+                ["N45.446241000", "W75.684339700"],
+                ["N45.445949000", "W75.684217200"],
+                ["N45.445652900", "W75.684116600"],
+                ["N45.445353600", "W75.684038100"],
+                ["N45.445051800", "W75.683981900"],
+                ["N45.444748400", "W75.683948200"],
+                ["N45.444444100", "W75.683936900"],
+                ["N45.444139900", "W75.683948200"],
+                ["N45.443836500", "W75.683982100"],
+                ["N45.443534800", "W75.684038400"],
+                ["N45.443235400", "W75.684117000"],
+                ["N45.442939300", "W75.684217700"],
+                ["N45.442647300", "W75.684340200"],
+                ["N45.442360100", "W75.684484200"],
+                ["N45.442078500", "W75.684649400"],
+                ["N45.441803400", "W75.684835300"],
+                ["N45.441535300", "W75.685041400"],
+                ["N45.441275200", "W75.685267100"],
+                ["N45.441023500", "W75.685512100"],
+                ["N45.440781200", "W75.685775300"],
+                ["N45.440548900", "W75.686056400"],
+                ["N45.440327100", "W75.686354400"],
+                ["N45.440116500", "W75.686668600"],
+                ["N45.439917800", "W75.686998100"],
+                ["N45.439731300", "W75.687342100"],
+                ["N45.439557800", "W75.687699600"],
+                ["N45.439397700", "W75.688069700"],
+                ["N45.439251400", "W75.688451200"],
+                ["N45.439119300", "W75.688843300"],
+                ["N45.439001800", "W75.689244800"],
+                ["N45.438899400", "W75.689654500"],
+                ["N45.438812100", "W75.690071400"],
+                ["N45.438740400", "W75.690494300"],
+                ["N45.438684500", "W75.690922000"],
+                ["N45.438644300", "W75.691353400"],
+                ["N45.438620200", "W75.691787200"],
+                ["N45.438612200", "W75.692222200"],
+                ["N45.438620200", "W75.692657200"],
+                ["N45.438644300", "W75.693091100"],
+                ["N45.438684400", "W75.693522400"],
+                ["N45.438740400", "W75.693950100"],
+                ["N45.438812200", "W75.694373100"],
+                ["N45.438899400", "W75.694789900"],
+                ["N45.439001800", "W75.695199700"],
+                ["N45.439119300", "W75.695601200"],
+                ["N45.439251400", "W75.695993200"],
+                ["N45.439397700", "W75.696374800"],
+                ["N45.439557800", "W75.696744800"],
+                ["N45.439731300", "W75.697102300"],
+                ["N45.439917800", "W75.697446300"],
+                ["N45.440116500", "W75.697775900"],
+                ["N45.440327100", "W75.6980
```

Comment:
please remove extra whitespace

Reply:
it keeps switching to line 1322 :thinking: the whitespace is on line 1806

Reply:
ah I got confused there